README.rst

```diff
@@ -95,6 +95,28 @@ Now install the script's dependencies:
 
     $ yum install python-markdown
 
+Upgrading OMERO.figure
+----------------------
+
+After upgrading OMERO.figure with:
+
+::
+
+    $ pip install -U omero-figure
+
+You need to update the Figure export script using one of the 2 options described
+above. If using *Option 1*, you need to *replace* the existing script:
```

Comment:
Maybe indicate that the user must have some admin privileges to do such operation.